phpcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<ruleset>
+    <file>src</file>
+    <file>tests</file>
+    <rule ref="PSR2"/>
```

Comment:
[PSR12](https://www.php-fig.org/psr/psr-12/) is the new - stricter - code style in town. Should we use that?

Reply:
Looks mostly like what we're already doing, so why not?

I don't like that `else` and `catch` should be on the same line as the previous closing brace, but that's probably due to too much Drupal exposure... I used to dislike the Drupal way.